src/Core/tests/DeviceTests/Handlers/HandlerTestBase.Tests.cs

```diff
@@ -47,7 +47,11 @@ public virtual async Task SetVisibility(Visibility visibility)
 			Assert.Equal(view.Visibility, id);
 		}
 
-		[Fact(DisplayName = "Semantic Description is set correctly")]
+		[Fact(DisplayName = "Semantic Description is set correctly"
+#if MONOANDROID
+			, Skip = "This value can't be validated through automated tests"
```

Comment:
Because we switched to using delegates for reading out the semantic description we can no longer verify these values through unit tests